cord19-explorer/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -4,6 +4,9 @@
 	   		<span sec:authorize="isAuthenticated()" onclick="openNav()" class="btn dark-btn"><i class="fa fa-bars"></i></span>	
 	      	<a th:href="@{/}" class="navbar-brand">CORD19 Geo Explorer</a>
 	   	</div>
+	   	<div sec:authorize="isAnonymous()" role="presentation">

```

Comment:
you don't need any authorization attribute. the way it's implemented right now, it would not show up, once you're logged in which doesn't make sense. Also, talk to @adhvikmadhav how to best integrate it into the layout.